openedx/core/djangoapps/content/learning_sequences/tasks.py

```diff
@@ -0,0 +1,101 @@
+import json
+import logging
+import pprint
+from collections import OrderedDict
+
+from celery.task import task
+from django.db import transaction
+from django.dispatch import receiver
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore import ModuleStoreEnum
+from xmodule.modulestore.django import modulestore, SignalHandler
+
+from .api import replace_course_outline
+from .api.data import (
+    CourseOutlineData, CourseItemVisibilityData, CourseSectionData, LearningSequenceData
+)
+
+
+@receiver(SignalHandler.course_published)
+def ls_listen_for_course_publish(sender, course_key, **kwargs):
```

Comment:
Note: Done inline as a temporary measure, because it's easier to debug when exceptions are raised.